examples/tree-view/src/Node/Node.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import {
+  addChild,
+  createNode,
+  deleteNode,
+  removeChild,
+  increment
+} from './treeSlice'
+
+const Node = React.memo(({ id, parentId }) => {
```

Comment:
:question: Not sure if passing an anonymous function to `React.memo()` this way correctly gives the nested component a display name in the DevTools. Can you check that?  If not, we'd want to extract this to its own separate variable first and then pass it to `memo(0`.

Reply:
You're correct, in the React Dev Tools, the Memo Component is labeled as Anonymous.

What should I name it? Is MemoNode okay?
```js
const MemoNode = React.memo(function Node({ id, parentId }) { ...
```

Reply:
That seems reasonable.